pubsub/google/cloud/pubsub/topic.py

```diff
@@ -426,11 +434,16 @@ class Batch(object):
                          before automatically commiting. Defaults to infinity
                          (off).
     :type max_messages: float
+
+    :param max_size: The maximum size that the serialized messages can be
+                     before automatically commiting. Defaults to just under
+                     10 MB.
+    :type max_size: int
     """
     _INFINITY = float('inf')
 
     def __init__(self, topic, client, max_interval=_INFINITY,
-                 max_messages=_INFINITY):
+                 max_messages=_INFINITY, max_size=10e6):
```

Comment:
1. You want `1e6`
1. `1e6` will be a float, not an integer, you almost certainly want an integer?